encryptedlogging/src/main/kotlin/com/automattic/encryptedlogging/store/EncryptedLogStore.kt

```diff
@@ -53,41 +42,21 @@ private const val MAX_RETRY_COUNT = 3
 private const val HTTP_STATUS_CODE_500 = 500
 private const val HTTP_STATUS_CODE_599 = 599
 
-internal class EncryptedLogStore constructor(
+internal class EncryptedLogStore(
```

Comment:
We'll have to assert, library-level, that this class will be a singleton as it holds a state 👍 .

Reply:
Aw nice, this is indeed a good catch, I agree, let's make it a singleton! 🔖